static/defaults.config.php

```diff
@@ -786,4 +786,9 @@
 		// Banner for Mastodon API
 		'mastodon_banner' => '/images/friendica-banner.jpg',
 	],
+	'blocklist' => [
+		// published (Boolean)
+		// Wether the blocklist is published under /about (or any later API)
+		'published' => true,
```

Comment:
After seeing the feedback about this list I don't think it should be published by default as it still isn't clear in Friendica that this list would be published as it is populated. I've also seen some flavor of Mastodon partially obfuscate the blocked server domain names to prevent easy search that can potentially lead to brigading or DoS attacks.

So I'd rather have a safer default.

Reply:
Before the change, the default was to show it. `$ git diff f20caf4c3a33025c7cfe799482697386bd9d4b8f src/Module/Friendica.php` shows this behavior. If we change it now to a "safer" one, most people won't notice it and maybe one day wonder why their blocklist isn't published suddenly anymore. I rather stay with the default and anyone who likes to hide it, can do it. Also I would expose this fact, e.g. when `blocklist.published` is set to false, then a message should appear telling this to users.

Reply:
And you still get DoS attacks for whatever reason, it is not directly related to publishing a blocklist. I don't see a direct and exclusive connection between these two. Plus I also want to hear other developer's opinions first.

Reply:
https://friendica.xyz/display/adf174d5-9364-a716-ba5c-ef2545253295 <-- that's what I mean with "other administrators expect the default behavior after update".

Reply:
At this point we're dealing with two distinct expectations:
- Admins who have known that the block list was published.
- Admins who didn't know that the block list was published and didn't expect it.

I'd rather satisfy the second group since it is better for privacy. The first group's expectation can still be met by changing this configuration value, while the second group won't have a bad surprise anymore.

Reply:
Okay, I can play along with it and change my mind about it. Do you still want to wait for @annando @nupplaphil @tobiasd @fabrixxm 's opinion on this?

Reply:
And what do you think about `blocklist.published`? I was thinking of `.enabled` instead, too.

Reply:
The blocklist will always be available for logged-in node users, so there's no complete disabling of it. Maybe change the name to include "public" if you aren't happy with ?